spec/injection_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'injection'
+
+describe Array do
+
+  describe 'injection' do
+    subject{[1,2,3]}
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.